plutus-contract/src/Plutus/Contract/Test.hs

```diff
@@ -184,6 +186,13 @@ defaultCheckOptions =
 changeInitialWalletValue :: Wallet -> (Value -> Value) -> CheckOptions -> CheckOptions
 changeInitialWalletValue wallet = over (emulatorConfig . initialChainState . _Left . ix wallet)
 
+allowBigTransactions :: CheckOptions -> CheckOptions
```

Comment:
Could add a comment explaining in which scenario the user might think of using this option? Like, what's the specific error one might get if the transaction is too big.